core/platforms/univention/CMakeLists.txt

```diff
@@ -57,7 +57,7 @@ install(
 )
 
 install(
-  FILES "bareos-com-release.key"
+  FILES "bareos-release.key"
```

Comment:
```suggestion
  # This file contains the Bareos repository signature public.
  # During the automatic build process, It will be added.
  # For manual builds, download the file from 
  # http://download.bareos.org/bareos/release/<RELEASE>/<DISTRIBUTION>/Release.key
  FILES "bareos-release.key"
```